Kickstarter-iOS/Views/Controllers/FeatureFlagToolsViewController.swift

```diff
@@ -35,7 +35,7 @@ final class FeatureFlagToolsViewController: UITableViewController {
     self.viewModel.outputs.reloadWithData
       .observeForUI()
       .observeValues { [weak self] features in
-        self?.features = features
+        self?.features = featureEnabledFromDictionaries(features)
```

Comment:
Hm, I wonder if it wouldn't be better if the viewModel returned the results of `featureEnabledFromDictionaries(features)`, so we wouldn't need to access its function directly. Is there any reason why you preferred this approach?

Reply:
Yeah this is intentional. The tests for this VC `FeatureFlagToolsViewControllerTests.swift` were testing that the ordering of the returned flags was correct. The problem (I felt) with those tests is that every time flags are removed those tests would break. When in fact that is not really what the tests were proving. So I changed the tests to just test simple dictionaries which meant that we needed an additional function to prove that we don't create `Feature` `enums` from flags that we don't support. For this purpose there is an [additional test](https://github.com/kickstarter/ios-oss/pull/1038/files#diff-ec92627879e52efe83c947c4f52aac09R102) which tests the function above.

Reply:
Oh ok. Thanks for explaining it.